roles/jd-server/src/lib/error.rs

```diff
@@ -8,8 +8,11 @@ use roles_logic_sv2::parsers::Mining;
 
 use crate::mempool::error::JdsMempoolError;
 
+pub type JdsResult<T> = core::result::Result<T, JdsError>;
```

Comment:
I see you didn't apply this to https://github.com/plebhash/stratum/pull/61

did you change your mind since writing this comment? or do you hold this as a suggestion for this current PR?